### PR TITLE
magento/magento2-page-builder#558: Developer can style content types …

### PR DESCRIPTION
**Pull request by hannahnida**
…output differently per viewport - Added body id in Storefront, added body id in tinymce4 in admin

### Description (*)

Changes are related to [Issue 558](https://github.com/magento/magento2-page-builder/issues/558), body id for storefront and for tinymce4 in admin area are necessary for the new styles to work properly.

### Related Pull Requests
https://github.com/magento/magento2-page-builder/pull/621
https://github.com/magento/magento2-page-builder-ee/pull/171
https://github.com/magento/partners-magento2ee/pull/387

### Manual testing scenarios (*)

1. ...
2. ...

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by hannahnida** [comment]
@magento run all tests

**Comment by hannahnida** [comment]
@magento run all tests

File: app/code/Magento/Theme/view/frontend/layout/default.xml [hunk 1]
```diff
@@ -8,6 +8,7 @@
 <page layout="3columns" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
     <update handle="default_head_blocks"/>
     <body>
+        <attribute name="id" value="html-body"/>
```
Comment (ihor-sviziev) [inline app/code/Magento/Theme/view/frontend/layout/default.xml:11]: @hannahnida I don't see what exactly should be done in the task you mentioned - but adding an ID to the body is not really seems correct solution here. Could you explain why it's needed? Why can't we use class here for instance? Unfortunately it's not clear what the styling issue you have

Comment (omiroshnichenko) [reply app/code/Magento/Theme/view/frontend/layout/default.xml:11]: @ihor-sviziev We need to add this id to have a higher level of CSS specificity over less styles. This work is going in the scope of Page Builder story to eliminate the usage of inline styles. Feel free to ask me for any details.

**Review by omiroshnichenko — APPROVED** [review]

**Comment by omiroshnichenko** [comment]
@magento run all tests

**Comment by omiroshnichenko** [comment]
@magento run all tests

**Comment by omiroshnichenko** [comment]
@magento run Performance Acceptance Tests

File: app/code/Magento/Theme/view/frontend/layout/default.xml (hunk 1 above)
Comment (ihor-sviziev) [reply app/code/Magento/Theme/view/frontend/layout/default.xml:11]: So why can't we use class here? I believe if you'll use `body.some-class .my-style` it should work approximately the same as `#my-body-id .my-style`

Comment (omiroshnichenko) [reply app/code/Magento/Theme/view/frontend/layout/default.xml:11]: Unfortunately, no. The class does not give enough specificity value. For more information, you could read https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity and visual specificity sheet https://specifishity.com/

**Comment by omiroshnichenko** [comment]
@magento run all tests

**Comment by omiroshnichenko** [comment]
@magento import pull request to magento-obsessive-owls/magento2ce

**Comment by magento-engcom-team** [comment]
@omiroshnichenko the pull request successfully imported.

**Pull request merged**